daemon/firewall/firewall_manager.go

```diff
@@ -0,0 +1,333 @@
+package firewall
+
+import (
+	"errors"
+	"fmt"
+	"net/netip"
+	"os/exec"
+	"sort"
+	"strings"
+
+	"github.com/NordSecurity/nordvpn-linux/daemon/device"
+)
+
+var ErrRuleAlreadyActive = errors.New("this rule is already active")
+
+const (
+	iptablesCommand  = "iptables"
+	ip6tablesCommand = "ip6tables"
+)
+
+// IptablesExecutor is an abstraction over iptables command. It should accept any input that is also viable for iptables
+// command.
+type IptablesExecutor interface {
+	InsertRule(command string) error
+	DeleteRule(command string) error
+	InsertRuleIPv6(command string) error
+	DeleteRuleIPv6(command string) error
+	// Enable changes the Executor state so that it performs the commands provided to ExecuteCommand and
+	// ExecuteCommandIPv6.
+	Enable()
+	// Disable changes the Executor state so that it does not perform commands provided to ExecuteCommands and
+	// ExecuteCommandIPv6.
+	Disable()
+}
+
+type Iptables struct {
```

Comment:
```suggestion
type IPTables struct {
```
And same for the all functions